app/views/plans/_edit_details.html.erb

```diff
@@ -49,6 +52,7 @@
           <%= f.label(:description, _('Project abstract'), class: 'control-label') %>
         </div>
         <div class="col-md-8" data-toggle="tooltip" title="<%= _("Briefly summarise your research project to help others understand the purposes for which the data are being collected or created.") %>">
+          <em class="sr-only"><%= _("Briefly summarise your research project to help others understand the purposes for which the data are being collected or created.") %></em>
```

Comment:
I think it would be better to set the string once and re-use the variable, like you've done in other places